presentation-api/receiving-ua/support/idlharness_receiving-ua.html

```diff
@@ -104,17 +106,21 @@
 </script>
```

Comment:
This does not appear in the diff, but as far as I can tell, the IDL here still has `PresentationConnectionClosedReason` instead of `PresentationConnectionCloseReason` (without `d`), and needs to be updated accordingly.